lib/apiservers/portlayer/swagger.json

```diff
@@ -59,6 +59,115 @@
 				}
 			}
 		},
+		"/kv": {
```

Comment:
@hickeng the api change to path is complete -- the switch to binary is not..how do you feel about tackling that after this?